src/test/java/teammates/test/cases/browsertests/FeedbackContributionQuestionUiTest.java

```diff
@@ -133,23 +152,54 @@ public void testDeleteQuestionAction() {
      * has options invalid for contribution questions. This is to prevent invalid options
      * from being copied over to the contribution question.
      */
-    private void testAddContributionQuestionAsSecondQuestion() {
```

Comment:
I found the previous name weird as we're not testing the adding per se, but the copying of the options under such a scenario. Will update the header comments!